lib/srv/app/server_test.go

```diff
@@ -38,10 +38,13 @@ import (
 	"github.com/gravitational/teleport/api/constants"
 	"github.com/gravitational/teleport/api/defaults"
 	"github.com/gravitational/teleport/api/types"
+	apievents "github.com/gravitational/teleport/api/types/events"
 	"github.com/gravitational/teleport/lib/auth"
 	"github.com/gravitational/teleport/lib/events"
 	"github.com/gravitational/teleport/lib/services"
+	libsession "github.com/gravitational/teleport/lib/session"
 	"github.com/gravitational/teleport/lib/utils"
+	"go.uber.org/atomic"
```

Comment:
Is this a new dependency? Can we use standard library instead?

Reply:
Not new. It is being used by the project already. I'm using it because I based another test, but I don't see any issue using the standard library.

Reply:
@gabrielcorado If we're already using it, it's fine.

Reply:
Nit: 
We are trying to keep import in following format where packages are split into 3  std/teleport/external sections: 
```
import (
  "path/filepath"
  "time"
  ...
  
  "github.com/gravitational/teleport/api/types"
  ...
  
  "go.uber.org/atomic"
  ... 
)
```